<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownLoadDialog</class>
 <widget class="QDialog" name="DownLoadDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pbClose">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>10</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="icon" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbTitle">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>下载管理</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>651</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="bottomWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>480</y>
     <width>651</width>
     <height>40</height>
    </rect>
   </property>
   <widget class="QToolButton" name="tbGoOn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tbClear">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
